source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -89,3 +89,9 @@ Removing a Library Dependency
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 To remove a library dependency from a project, select **Manage Current Libraries** from the **Manage Vendor Libraries** menu, check the box for any libraries to uninstall and click OK. These libraries will be removed as dependencies from the project.
+
+Command-Line Interface (Online)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Adding a vendor library dependency from the vendor URL can also be done through the command-line via a gradle task. Open a command-line instance at the project root, and enter `gradlew vendordep --url=<url>` where `<url>` is the vendor JSON URL. This will add the vendor library dependency JSON file to the **vendordeps** folder of the project. Vendor libraries can be updated the same way.
```

Comment:
```suggestion
Adding a vendor library dependency from the vendor URL can also be done through the command-line via a gradle task. Open a command-line instance at the project root, and enter ``gradlew vendordep --url=<url>`` where ``<url>`` is the vendor JSON URL. This will add the vendor library dependency JSON file to the ``vendordeps`` folder of the project. Vendor libraries can be updated the same way.
```